cfg_mgmt/reporting.py

```diff
@@ -86,107 +84,6 @@ def evaluate_cfg_element_status(
     )
 
 
-def cfg_element_statuses_responsible_summaries(
```

Comment:
how is removal of elasticsearch related to removal of this piece of code? is this a left-over that we kept for the sole purpose of also uploading to elasticsearch?

Reply:
yes, there are some dedicated functions for cfg-status upload towards elasticsearch.
this has **nothing** to do with github compliance issues for cfg elements